nominees/opensourceecology.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "OpenSourceEcology",
+  "description": "Creating and sharing open source industrial machine designs",
+  "website": "https://www.opensourceecology.org/",
+  "license": [
+    {
+      "spdx": "GPL-3.0",
+      "licenseURL": "https://github.com/OpenSourceEcology/cross-platform-python-gui"
```

Comment:
@Msgl kindly update the `licenseURL` to point to a specific page within the same repo referencing the [license](https://github.com/OpenSourceEcology/cross-platform-python-gui/blob/master/LICENSE) . Thanks.